src/DocRaptor.php

```diff
@@ -71,10 +67,19 @@ public function setDocumentUrl($document_url)
     /**
      * @param string $document_type
      * @return $this
+     * @throws Exception
      */
     public function setDocumentType($document_type)
     {
+        $allowedValues = array('xls', 'xlsx', 'pdf');
```

Comment:
Adoption is pretty slow but many will skip right to PHP 7 when adoption should speed up (I hope). Let's leave it for the moment.